packages/darklang/languageTools/packageManager.dark

```diff
@@ -30,4 +30,14 @@ module Darklang =
           Builtin.packageManagerFindFn name
 
         let get (id: Uuid) : Stdlib.Option.Option<ProgramTypes.PackageFn.PackageFn> =
-          Builtin.packageManagerGetFn id
\ No newline at end of file
+          Builtin.packageManagerGetFn id
+
+
+      let pm () : ProgramTypes.PackageManager.PackageManager =
```

Comment:
let's add a CLEANUP here for this to be a constant/value, rather than a fn, once we can have constant records